### PR TITLE
SCAL-77285-origin-param-for-spotdev-btn

### PR DESCRIPTION
**Pull request by pankajp-ts**
Ticket - https://thoughtspot.atlassian.net/browse/SCAL-77285

### CODE REVIEW

**Comment by pankajp-ts** [comment]
> Ticket - https://thoughtspot.atlassian.net/browse/SCAL-77285

@ayonghosh moved all files and folder from `assets` to `static` as per GatsBy asciidoc way

File: docs/src/components/LeftSidebar/index.tsx [hunk 1]
```diff
@@ -2,9 +2,18 @@ import React from 'react';
 import './index.scss';
 import BackButton from '../BackButton';
 
-const LeftSideBar = (props: { navTitle: string; navContent: string }) => (
+const LeftSideBar = (props: {
+    navTitle: string;
+    navContent: string;
+    spotDevHomeLink: string;
```
Comment (ayonghosh) [inline docs/src/components/LeftSidebar/index.tsx:8]: Should we just call this backLink?

File: docs/src/components/LeftSidebar/index.tsx [hunk 2]
```diff
@@ -2,9 +2,18 @@ import React from 'react';
...
+    navTitle: string;
+    navContent: string;
+    spotDevHomeLink: string;
+}) => (
     <aside>
-        <BackButton title="SpotDev Home" />
+        {props.spotDevHomeLink && (
+            <BackButton
+                title="SpotDev Home"
+                spotDevHomeLink={props.spotDevHomeLink}
```
Comment (ayonghosh) [inline docs/src/components/LeftSidebar/index.tsx:14]: Same here: backLink

**Review by ayonghosh — COMMENTED** [review]
LGTM after addressing comments

**Review by ayonghosh — APPROVED** [review]

**Pull request merged**